site/content/en/docs/Reference/fleet.md

```diff
@@ -129,7 +150,6 @@ A `FleetAllocation` is used to allocate a `GameServer` out of an existing `Fleet
 A full `FleetAllocation` specification is available below and in the 
 {{< ghlink href="examples/fleetallocation.yaml" >}}example folder{{< /ghlink >}} for reference:
 
-{{% feature expiryVersion="0.12.0" %}}
```

Comment:
Why did you remove this part?

Reply:
You can see it was second and redundant part, it has one closing tag and two expiryVersion above.

Reply:
line 121 {{% feature expiryVersion="0.12.0" %}}
## Fleet Allocation Specification

Reply:
https://github.com/googleforgames/agones/blame/0b6749d69fbcd23384aaf42f6a61aa3740c5e380/site/content/en/docs/Reference/fleet.md#L121
I cannot find the exact commit were it happens. But as you could see from link above.
There are two opening `expiryVersion`  with one closing.
Also at https://github.com/googleforgames/agones/blame/0b6749d69fbcd23384aaf42f6a61aa3740c5e380/site/content/en/docs/Reference/fleet.md#L132

Reply:
So there were a nested feature blocks. And as long we don't need FleetAllocation any more we don't need to publish anything about it.

Reply:
makes sense, thanks. that was probably my doing, so thank you for fixing it.